packages/app/src/dimension-mapper/SlicingSlider.module.css

```diff
@@ -28,7 +28,7 @@
 }
 
 .track {
-  flex: 1 0 auto;
```

Comment:
This used to be a workaround to `1 1 0%` for flex items inside column flex containers in legacy browsers. Just like `0%`, it is no longer relevant.